en/ability.json

```
@@ -1,99 +1,99 @@
 {
   "stench": {
     "name": "Stench",
-    "description": "By releasing stench when attacking, this Pokémon may cause the target to flinch."
+    "description": "By releasing a stench when attacking, this Pokémon has a 10% chance to flinch the target. This does not stack with moves that already flinch."
   },
   "drizzle": {
     "name": "Drizzle",
     "description": "The Pokémon makes it rain when it enters a battle."
   },
   "speedBoost": {
     "name": "Speed Boost",
-    "description": "Its Speed stat is boosted every turn."
+    "description": "The Pokémon’s Speed stat is boosted every turn."
   },
   "battleArmor": {
     "name": "Battle Armor",
     "description": "Hard armor protects the Pokémon from critical hits."
   },
   "sturdy": {
     "name": "Sturdy",
-    "description": "It cannot be knocked out with one hit. One-hit KO moves cannot knock it out, either."
+    "description": "The Pokémon cannot be knocked out by a single hit as long as its HP is full. One-hit KO moves will also fail to knock it out."
   },
   "damp": {
     "name": "Damp",
-    "description": "Prevents the use of explosive moves, such as Self-Destruct, by dampening its surroundings."
+    "description": "The Pokémon dampens its surroundings, preventing all Pokémon from using explosive moves such as Self-Destruct or dealing damage with Aftermath."
   },
   "limber": {
     "name": "Limber",
-    "description": "Its limber body protects the Pokémon from paralysis."
+    "description": "The Pokémon’s limber body prevents it from being paralyzed."
   },
   "sandVeil": {
     "name": "Sand Veil",
-    "description": "Boosts the Pokémon’s evasiveness in a sandstorm."
+    "description": "Boosts the Pokémon’s evasiveness in a sandstorm by 20%. It also takes no damage from sandstorms."
   },
   "static": {
     "name": "Static",
-    "description": "The Pokémon is charged with static electricity, so contact with it may cause paralysis."
+    "description": "Contact with the Pokémon has a 30% chance to paralyze the attacker."
```

```suggestion
    "description": "The Pokémon is charged with static electricity and has a 30% chance to paralyze attackers that make direct contact with it."
```
Gen 9 description with chance specified